ts/torch_handler/base_handler.py

```diff
@@ -53,6 +53,10 @@
     )
     PT2_AVAILABLE = False
 
+if packaging.version.parse(torch.__version__) > packaging.version.parse("2.1.1"):
```

Comment:
i think packaging throws a bunch of warnings right? Can we do whatever the warning recommends cause our logs are starting to get noisy

Reply:
I don't see any warnings from packaging